### PR TITLE
swift 3.0 ready

### PR DESCRIPTION
**Pull request by adriencanterot**
Ready for 1.0

### CODE REVIEW

**Comment by adriencanterot** [comment]
Sorry for the conflicts, I forgot you had made some changes too, should I create a new PR or can you easily resolve them ? Thanks

**Comment by adriencanterot** [comment]
Alright, managed to do it by myself, now ready for 1.0 unless you want to add a 1.0 tag to csqlite first @tannernelson

**Comment by tannernelson** [comment]
Just pushed CSQLite 1.0

File: Package.swift [hunk 1]
```diff
@@ -2,8 +2,9 @@ import PackageDescription
 
 let package = Package(
     name: "SQLite",
-    dependencies: [ 
+    dependencies: [
         .Package(url: "https://github.com/vapor/csqlite.git", majorVersion: 0, minor: 1),
```
Comment (tannernelson) [inline Package.swift]: `..., majorVersion: 1)`

**Comment by adriencanterot** [comment]
Ready for 1.0 !

**Pull request merged**